notebook/static/notebook/js/actions.js

```diff
@@ -545,13 +552,13 @@ define(function(require){
         },
     };
 
-    // private stuff that prepend `.ipython` to actions names
+    // private stuff that prepend `jupyter-notebook:` to actions names
     // and uniformize/fill in missing pieces in of an action.
     var _prepare_handler = function(registry, subkey, source){
-        registry['ipython.'+subkey] = {};
-        registry['ipython.'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
-        registry['ipython.'+subkey].help_index = source[subkey].help_index;
-        registry['ipython.'+subkey].icon = source[subkey].icon;
+        registry['jupyter-notebook:'+subkey] = {};
+        registry['jupyter-notebook:'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
+        registry['jupyter-notebook:'+subkey].help_index = source[subkey].help_index;
+        registry['jupyter-notebook:'+subkey].icon = source[subkey].icon;
```

Comment:
It seems weird that we would namespace `jupyter-notebook` in our own Jupyter-specific command namespace when we felt `notebook` was a sufficiently unique key in the entire namespace of Python packages. Seems like either `notebook` is okay here or `notebook` wasn't an okay choice for the Python package name.